### PR TITLE
docs(PFD Briefing): corrections and updates

### LINKED ISSUE #251: PFD Briefing Corrections - Review in #117
Simple typos.  Looks like one too many ~ for VMO (for subscript) and missing subscript entirely for MMO.

_Originally posted by @donstim in https://github.com/flybywiresim/docs/pull/177#discussion_r725241481_

Other comments in https://github.com/flybywiresim/docs/pull/177

### PR DESCRIPTION
**Pull request by Valastiri**
fixes #251

## Summary

Currently only targeting review comments made in https://github.com/flybywiresim/docs/pull/177 past the merge date. 

A rather large change to the V2 speed description to provide more clarity and detail to engine out ops and general definition of V2.

image

### Location

 docs/pilots-corner/a32nx-briefing/pfd/speedtape.md

Discord username (if different from GitHub):
 Valstiri#8902

### CODE REVIEW

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md [hunk 1]
```diff
@@ -49,13 +49,13 @@ This is displayed in green and is shown if the Mach number reaches above 0.5. It
 
 This is displayed in green and shows the speed at which overspeed protections become active.
 
-This speed is V~MO~~ (maximum operating speed) + 6 knots or MMO (maximum Mach speed) + 0.01
+This speed is V~MO~ (maximum operating speed) + 6 knots or ~MMO~ (maximum Mach speed) + 0.01
```
Comment (frankkopp) [inline docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]:
```suggestion
This speed is V~MO~ (maximum operating speed) + 6 knots or M~MO~ (maximum Mach speed) + 0.01
```

Only subscripted when added to Vxyz

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md [hunk 2]
```diff
@@ -49,13 +49,13 @@ This is displayed in green and is shown if the Mach number reaches above 0.5. It
...
 
 ### ECON Speed Range
 
 This appears in descent mode with ECON/AUTO SPD mode active. Two half triangles will represent the upper and lower limits calculated by the FMGC. This replaces the target speed symbol.
 
-The upper speed is equal to the target speed + 20 knots, limited to V~MAX~, V~MO~ - 3 knots or MMO - 0.006, whichever is the lowest.
+The upper speed is equal to the target speed + 20 knots, limited to V~MAX~, V~MO~ - 3 knots or ~MMO~ - 0.006, whichever is the lowest.
```
Comment (frankkopp) [inline docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]:
```suggestion
The upper speed is equal to the target speed + 20 knots, limited to V~MAX~, V~MO~ - 3 knots or M~MO~ - 0.006, whichever is the lowest.
```

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ The top of a red strip along the speed scale indicates this speed. It represents
 
 The lower end of a red and black strip along the speed scale represents this speed. It is the lowest of:
 
-- V~MO~ or the corresponding to MMO
+- V~MO~ or the corresponding to ~MMO~
```
Comment (frankkopp) [inline docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]:
```suggestion
- V~MO~ or the corresponding to M~MO~
```

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md [hunk 4]
```diff
@@ -110,7 +110,14 @@ This is a blue dot on the right side of the speedtape. It displays the speed at
 
 ### V~2~
 
-This is a magenta triangle on the right side of the speedtape. It displays the V~2~ speed which has been set in the TAKE OFF PERF page of the MCDU. This is the speed the aircraft needs to maintain at a minimum up to the acceleration altitude.
+This is a magenta triangle on the right side of the speedtape. It displays the V~2~ speed which has been set in the TAKE OFF PERF page of the MCDU. V~2~ is the minimum speed that needs to be maintained in the even to of an engine failure after V~1~ up to acceleration altitude.
```
Comment (frankkopp) [inline docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]:
```suggestion
This is a magenta triangle on the right side of the speedtape. It displays the V~2~ speed which has been set in the TAKE OFF PERF page of the MCDU. V~2~ is the minimum speed that needs to be maintained in the event of an engine failure after V~1~ up to acceleration altitude.
```

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md [hunk 5]
```diff
@@ -110,7 +110,14 @@ This is a blue dot on the right side of the speedtape. It displays the speed at
...
+
+V~2~+10 provides the best climb performance (before acceleration altitude + during NAPs).
```
Comment (frankkopp) [inline docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: reference?

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md [hunk 6]
```diff
@@ -110,7 +110,14 @@ This is a blue dot on the right side of the speedtape. It displays the speed at
...
+
+!!! info "Engine Loss V~2~ Speeds"
+    After takeoff **V~2~+10** is the targeted speed you should reach and maintain by the time you attain a height of 35 FT after takeoff with an engine failure until reaching the acceleration altitude. Pilots should not degrade to V~2~ after an engine failure but rather maintain the speed at point of engine loss where V~2~ is the minimum allowable speed.
```
Comment (frankkopp) [inline docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: Reference for V2:
image

V2+10 reference?

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md (hunk 6 above)
Comment (frankkopp) [reply docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: image

Comment (frankkopp) [reply docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: image

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md (hunk 2 above)
Comment (frankkopp) [reply docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: Reference: Wikipedia:
image

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md (hunk 5 above)
Comment (donstim) [reply docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: V2+10 provide better climb performance, not the "best" climb performance. A higher speed would provide even better climb performance, but it could negatively affect close-in obstacle clearance if it requires a lower climb angle while accelerating to that speed.

I recommend writing out Noise Abatement Program Departures rather than "NAPs" as I doubt many readers of this material will have a clue as what NAPs refers to. It isn't even in the list of abbreviations that may in itself be hard to find.

File: docs/pilots-corner/a32nx-briefing/pfd/speedtape.md [hunk 7]
```diff
@@ -110,7 +110,14 @@ This is a blue dot on the right side of the speedtape. It displays the speed at
 
 ### V~2~
 
-This is a magenta triangle on the right side of the speedtape. It displays the V~2~ speed which has been set in the TAKE OFF PERF page of the MCDU. This is the speed the aircraft needs to maintain at a minimum up to the acceleration altitude.
+This is a magenta triangle on the right side of the speedtape. It displays the V~2~ speed which has been set in the TAKE OFF PERF page of the MCDU. V~2~ is the minimum speed that needs to be maintained in the event of an engine failure after V~1~ up to acceleration altitude.
+
+V~2~+10 provides the best climb performance (before acceleration altitude + during NAPs).
+
+!!! info "Engine Loss V~2~ Speeds"
+    After takeoff **V~2~+10** is the targeted speed you should reach and maintain by the time you attain a height of 35 FT after takeoff with an engine failure until reaching the acceleration altitude. Pilots should not degrade to V~2~ after an engine failure but rather maintain the speed at point of engine loss where V~2~ is the minimum allowable speed.
```
Comment (donstim) [inline docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: Suggest pilots should not "decelerate to" rather than not "degrade to." And may as well add "but not higher than V2+15" to the "maintain the speed at the time of engine failure" (a bit of suggest re-wording there as well). I assume you saw my further response to this item in the old PR so I don't have to reproduce it here.

EDIT:  I have a further comment. The current text you have says you should maintain V2+10 after takeoff with an engine failure. V2+10 is the speed you should reach without an engine failure.  The target speed for the engine failure case depends on when the engine failure occurs. If it is before reaching V2 (like before or just after lifting off), the target is V2. If after you are at a speed higher than V2, then you maintain the higher speed (but no higher than V2+15).

Comment (Valastiri) [reply docs/pilots-corner/a32nx-briefing/pfd/speedtape.md]: Perfect thanks, I'll reword this to take all into account, was waiting for input and referential information to close both urs and mav's comments.

**Review by frankkopp — APPROVED** [review]

**Comment by Valastiri** [comment]
@donstim Would you like to continue review on PFD or are you happy we merge this?

You could open an issue with any changes to production as well. Let me know :)

**Comment by donstim** [comment]
LGTM
